releasetasks/templates/firefox/funsize.yml.tmpl

```diff
@@ -0,0 +1,113 @@
+# keep first line
+-
+    taskId: "{{ stableSlugId("funsize_update_generator_image") }}"
```

Comment:
maybe this filename should be funsize_image.yml.tmpl so it doesn't cause confusion